k8s/charts/openebs/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v1
-version: 0.8.2
+version: 0.9.0
 name: openebs
-appVersion: 0.8.1
+appVersion: 0.9.0
 description: Containerized Storage for Containers
-icon: https://raw.githubusercontent.com/openebs/chitrakala/master/OpenEBS%20logo/openebs%20logos-03.png
+icon: https://raw.githubusercontent.com/openebs/chitrakala/master/OpenEBS%20logo/logo-square.png
```

Comment:
Change the link to : https://raw.githubusercontent.com/cncf/artwork/master/projects/openebs/icon/color/openebs-icon-color.png

Reply:
done